src/lib/extensions/dashboard/DashboardExtension.ts

```diff
@@ -11,11 +13,16 @@ export class DashboardExtension<ParamType extends Params = Params> extends Exten
    */
   public params!: ParamType;
   /**
+   * Deprecated - has been superseded by the @hub object.
```

Comment:
You can use a JSDoc property to mark as deprecated

https://jsdoc.app/tags-deprecated.html